ci/.gitlab-ci.yml

```diff
@@ -30,43 +37,25 @@ cpu debug build:
   extends: .build_common
   variables:
     BUILD_DOCKER_FILE: ci/docker/debug/build.Dockerfile
+    DEPLOY_DOCKER_FILE: ci/docker/debug/deploy.Dockerfile
     BUILD_IMAGE_BASE: "$CI_REGISTRY_IMAGE/debug/build:"
     IMAGE: $CI_REGISTRY_IMAGE/debug/deploy2:$CI_COMMIT_SHA
```

Comment:
@rasolca Is `debug/deploy2` name still problematic?

Reply:
`$CI_REGISTRY_IMAGE/debug/deploy` has still problems. I reduced the number of images but it is still not navigable in gitlab.